src/main/java/com/minecolonies/coremod/client/gui/WindowBuildBuilding.java

```diff
@@ -0,0 +1,362 @@
+package com.minecolonies.coremod.client.gui;
+
+import com.minecolonies.api.crafting.ItemStorage;
+import com.minecolonies.api.util.BlockPosUtil;
+import com.minecolonies.api.util.BlockUtils;
+import com.minecolonies.api.util.ItemStackUtils;
+import com.minecolonies.api.util.LanguageHandler;
+import com.minecolonies.api.util.constant.ColorConstants;
+import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.blockout.Pane;
+import com.minecolonies.blockout.controls.Button;
+import com.minecolonies.blockout.controls.ButtonHandler;
+import com.minecolonies.blockout.controls.ItemIcon;
+import com.minecolonies.blockout.controls.Label;
+import com.minecolonies.blockout.views.DropDownList;
+import com.minecolonies.blockout.views.ScrollingList;
+import com.minecolonies.blockout.views.Window;
+import com.minecolonies.coremod.MineColonies;
+import com.minecolonies.coremod.blocks.AbstractBlockHut;
+import com.minecolonies.coremod.blocks.BlockSolidSubstitution;
+import com.minecolonies.coremod.blocks.ModBlocks;
+import com.minecolonies.coremod.colony.CitizenDataView;
+import com.minecolonies.coremod.colony.ColonyView;
+import com.minecolonies.coremod.colony.StructureName;
+import com.minecolonies.coremod.colony.Structures;
+import com.minecolonies.coremod.colony.buildings.BuildingBuilder;
+import com.minecolonies.coremod.colony.buildings.utils.BuildingBuilderResource;
+import com.minecolonies.coremod.colony.buildings.views.AbstractBuildingView;
+import com.minecolonies.coremod.entity.ai.basic.AbstractEntityAIStructure;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.entity.ai.citizen.builder.EntityAIStructureBuilder'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)